Detectors/TOF/base/src/CalibTOFapi.cxx

```diff
@@ -41,13 +44,42 @@ CalibTOFapi::CalibTOFapi(const std::string url)
 
 //______________________________________________________________________
 
+void CalibTOFapi::readActiveMap()
+{
+  // getting the active TOF map
+  memset(mIsOffCh, false, Geo::NCHANNELS);
+
+  auto& mgr = CcdbManager::instance();
+  long timems = long(mTimeStamp) * 1000;
```

Comment:
I guess you referred to
-  corr += mLHCphase->getLHCphase(int(mTimeStamp / 1000)); // timestamp that we use in LHCPhase is in seconds, but for CCDB we need it in ms
+  corr += mLHCphase->getLHCphase(mTimeStamp); // timestamp that we use in LHCPhase is in seconds, but for CCDB we need it in ms

since LHCphase get it in second (fixed)

Reply:
mTimeStamp -> seconds

Reply:
ok, did not notice that the 2 nd call is not a ccdb query.

Reply:
Yes, we planned in the past to have an internal structure here.
However, we receently moved to use calibration timeslot to fill ccdb so this is probably an obsolete feature (but we will decide to keep one ccdb object per run with the LHCphase trending, this should be discussed).